content/automate/ManageIQ/System/Event/EmsEvent/Hawkular.class/hawkular_jdbc.ok.yaml

```diff
@@ -0,0 +1,12 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: JDBC Success
```

Comment:
missing a verb/action here. what does it mean JDBC success?

Reply:
I've tried to follow what we already have [here](https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/System/Event/EmsEvent/Hawkular.class/hawkular_deployment.ok.yaml) (I considered Deployment as a noun not a verb, my bad). I'm gonna change it so we're more consistent with the https://github.com/ManageIQ/manageiq/pull/16134.